etc/init.d/contextBroker.ubuntu

```diff
@@ -0,0 +1,184 @@
+#! /bin/sh
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
```

Comment:
Seeing virtual box init script:

#! /bin/sh
# Sun VirtualBox
# Linux kernel module init script

#
# Copyright (C) 2006-2009 Sun Microsystems, Inc.
#
# This file is part of VirtualBox Open Source Edition (OSE), as
# available from http://www.virtualbox.org. This file is free software;
# you can redistribute it and/or modify it under the terms of the GNU
# General Public License (GPL) as published by the Free Software
# Foundation, in version 2 as it comes in the "COPYING" file of the
# VirtualBox OSE distribution. VirtualBox OSE is distributed in the
# hope that it will be useful, but WITHOUT ANY WARRANTY of any kind.
#

# chkconfig: 35 30 70
# description: VirtualBox Linux kernel module
#
### BEGIN INIT INFO
# Provides:       vboxdrv virtualbox
# Required-Start: $remote_fs $network
# Required-Stop:  $remote_fs
# Default-Start:  2 3 4 5
# Default-Stop:   0 1 6
# Short-Description: VirtualBox Linux kernel module
### END INIT INFO


So the license is before, im going to revert the last commit.

2015-01-29 9:56 GMT+01:00 Fermín Galán Márquez <notifications@github.com>:

> In etc/init.d/contextBroker.ubuntu
> <https://github.com/telefonicaid/fiware-orion/pull/679#discussion_r23753658>
> :
>
> > @@ -0,0 +1,184 @@
> > +#! /bin/sh
> > +# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
> > +#
> > +# This file is part of Orion Context Broker.
>
> Digging into this, it seems that the official ocumentation about the
> BEGING/END INIT INFO system doesn't say anything about this (see
> http://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/initscrcomconv.html
> )
>
> However, looking to some random examples, it seems to be ok having the
> license header before this block (as it is now in your code).
>
>
> https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1/src/packages/deb/init.d/hadoop-jobtracker
>
> www.virtualbox.org/svn/vbox/trunk/src/VBox/ValidationKit/utils/TestExecServ/linux/vboxtxs.sh
>
> So, I'd suggest to have a look into the file in /etc/init.d directory in
> your Ubuntu system (I'm afraid I cannot do it myself, as I don't use Ubuntu
> :)) and, if they are using the same pattern, let as it it now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/telefonicaid/fiware-orion/pull/679/files#r23753658>.
>